foundationdb/src/main/java/site/ycsb/db/foundationdb/FoundationDBClient.java

```diff
@@ -57,6 +60,14 @@
   private static final String EXTERNAL_CLIENT_DIRECTORY_DEFAULT = ".";
   private static final String TRACE_DIRECTORY            = "foundationdb.tracedirectory";
   private static final String TRACE_DIRECTORY_DEFAULT    = "";
+  private static final String SET_TRANSACTION_TRACE      = "foundationdb.settransactiontrace";
+  private static final String SET_TRANSACTION_TRACE_DEFAULT = "";
+  private static final String TRANSACTION_TRACE_FRACTION = "foundationdb.transactiontracefraction";
+  private static final String TRANSACTION_TRACE_FRACTION_DEFAULT = "10000";
```

Comment:
"Fractions" above 1 are strange.  Use `parseDouble` and default this to `0.0001` (and get rid of the reciprocal below).